clmm/utils.py

```diff
@@ -668,20 +668,22 @@ def compute_beta(z_src, z_cl, cosmo):
 
     Parameters
     ----------
-    z_src:  float
-        Source galaxy redshift
+    z_src:  float, array_like
+        Source galaxies redshift
     z_cl: float
         Galaxy cluster redshift
     cosmo: clmm.Cosmology
         CLMM Cosmology object
 
     Returns
     -------
-    float
-        Geometric lensing efficicency
+    numpy array
+        Geometric lensing efficicency for each source
     """
-    beta = np.heaviside(z_src-z_cl, 0) * cosmo.eval_da_z1z2(z_cl, z_src) / cosmo.eval_da(z_src)
-    return beta
+    if type(z_src) is not list:
+        z_src = [z_src]
```

Comment:
I don't know if I should keep this test or expect the user to pass a list with just one source. @m-aguena